storage/store/inmemory/unsynchronized/registers.go

```diff
@@ -0,0 +1,114 @@
+package unsynchronized
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/bits-and-blooms/bloom/v3"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+type RegisterEntries map[flow.RegisterID]flow.RegisterValue
+type HeightToRegisterEntries map[uint64]RegisterEntries
```

Comment:
since this storage object will only contain data for a single block, you can skip the map keyed on block height.